### PR TITLE
🎯 Improve Flash Messages 🎨

### PR DESCRIPTION
**Pull request by shrihari689**
## Proposed Changes 🔥

- Redesign Flash Messages 🖌
- Rename Flash Message Names to follow [convenience accessors](https://api.rubyonrails.org/classes/ActionDispatch/Flash.html) ✏
- Fixed Logout Button 🔨
- Add Remove Button for Flash ❌

## Merge Checklist ✅

- [x] ~~Add specs that demonstrate bug / test a new feature.~~
- [x] ~~Check if route, query, or mutation authorization looks correct.~~
  - ~~Add tests for authorization, if required.~~
- [x] Update docs, where applicable.
- [x] ~~Add development seeds for new tables.~~

Followed by #58 📌 Reported by @peeyush14goyal 📍

### CODE REVIEW

**Review by abhinandanarya06 — APPROVED** [review]
Looks good to me. Hope the problem in main branch will resolve as soon as possible. Appreciate your hard work and dedication

File: app/javascript/layouts/sidebar/LayoutSidebar__Root.res [hunk 1]
```diff
@@ -37,12 +39,12 @@ let userDetails = currentUserName => {
           <p className="text-base font-medium text-gray-700 group-hover:text-gray-900">
             {s(currentUserName)}
           </p>
-          <p className="text-sm font-medium text-gray-500 group-hover:text-gray-700">
+          <button className="text-sm font-medium text-gray-500 group-hover:text-gray-700">
```
Comment (peeyush14goyal) [inline app/javascript/layouts/sidebar/LayoutSidebar__Root.res:42]: Is this optimized for mobile view also ?

Comment (shrihari689) [reply app/javascript/layouts/sidebar/LayoutSidebar__Root.res:42]: Yeah 🎉. As it was. 📍

**Comment by shrihari689** [comment]
Things to be noted: 
 - Use `flash[:alert]` and `flash[:notice]` hereafter
 - Or can also use this [method](https://api.rubyonrails.org/classes/ActionController/Redirecting.html#method-i-redirect_to)
 - Have a look at this UI/UX style for Buttons. 27be22aa

 📌

**Pull request merged**